openflow13/bundles.go

```diff
@@ -149,7 +149,8 @@ func (p *BundlePropertyExperimenter) UnmarshalBinary(data []byte) error {
 	p.ExperimenterType = binary.BigEndian.Uint32(data[n:])
 	n += 4
 	if len(data) < int(p.Length) {
```

Comment:
I am having a hard time understanding this code. I took a look at the Openflow spec and we should always have `len(data) >= int(p.Length)`, i.e., the length of the message should be at least equal to the property length (which includes header + experimenter data). What am I missing?

Reply:
Good catch, it should be an issue existing for long. I think here should be `if n < int(p.Length)`